packages/coachmark/src/Coachmark.ts

```diff
@@ -1,44 +1,92 @@
 /*
-Copyright 2020 Adobe. All rights reserved.
+Copyright 2023 Adobe. All rights reserved.
 This file is licensed to you under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License. You may obtain a copy
 of the License at http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software distributed under
 the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
 
 import {
-    CSSResultArray,
     html,
     SpectrumElement,
     TemplateResult,
 } from '@spectrum-web-components/base';
-import { property } from '@spectrum-web-components/base/src/decorators.js';
-
-import styles from './coachmark.css.js';
+import type { Overlay, Placement } from '@spectrum-web-components/overlay';
+import { property, state } from 'lit/decorators.js';
+import type { CoachmarkItem } from './CoachmarkItem.js';
+import './CoachmarkPopover.js';
+import '@spectrum-web-components/overlay/sp-overlay.js';
 
 /**
  * @element sp-coachmark
  */
 export class Coachmark extends SpectrumElement {
```

Comment:
Does this element specifically save a consumer anything over them using an `<sp-overlay>` directly?

Reply:
I mean to say, “why this element at all?”. It seems like it’s a wrapper on `sp-overlay` and I’m not sure what you get for it other than additional shadow boundaries, prop drilling, and intermediate state containers.